VAMobile/documentation/docs/Engineering/DevOps/Automation Code Docs/GitHub Actions/ReleaseTicketIssues.md

```diff
@@ -0,0 +1,26 @@
+# Release Tickets Issues
```

Comment:
Can this content be moved to the "Release Issues" page we already have? https://department-of-veterans-affairs.github.io/va-mobile-app/docs/Engineering/DevOps/Automation%20Code%20Docs/GitHub%20Actions/ReleaseIssues